proposals/collection-expression-arguments.md

```diff
@@ -111,9 +111,9 @@ passed to that constructor, which is definitely is not.
 Based on all of the above, a small handful of options have come up that are felt to solve the needs of passing
 arguments, without stepping out of bounds of the goals of collection expressions.
 
-## Option 1: `[with(...arguments...)]`
+## `[with(...arguments...)]` Design
 
-The design of this form would be as follows:
+Syntax:
```

Comment:
From the link you'd sent me, it sounds like the constructor section needs to be revised and a new proposal offered. [https://github.com/dotnet/csharplang/blob/main/meetings/2025/LDM-2025-05-07.md#collection-expression-arguments](https://github.com/dotnet/csharplang/blob/main/meetings/2025/LDM-2025-05-07.md#collection-expression-arguments)
It would be good to review the LDM notes and reflect decisions somehow in the spec (empty arg list, 

How do you want to track open issues? FWIW, I think the format used for extensions feature worked out well ( [https://github.com/dotnet/csharplang/blob/main/proposals/csharp-14.0/extensions.md](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-14.0/extensions.md) )